src/main/java/com/lofri/catchtable/common/code/ResponseCode.java

```diff
@@ -28,6 +30,18 @@ public enum ResponseCode {
             new Class[]{
             DuplicateEmailException.class}),
 
+    C200_001("Duplicate nickname",
+            HttpStatus.BAD_REQUEST,
+            new Class[] {
+                    DuplicateNicknameException.class
+            }),
```

Comment:
DUPLICATE_NICKNAME("200_001", "nickname is duplicated", HttpStatus.BAD_REQUEST, ...)